### ISSUE TITLE
Configuration should be in a PHP file, not INI file.

### ISSUE THREAD

**Issue by Alanaktion** [Alanaktion/phproject#215]
Currently the INI file is readable by default on most web servers. That's bad. We should move it to a PHP file to ensure no one accidentally leaks their database credentials.

Something like this would work great:

``` php
// config.php
return array(
    "db" => array(
        "host" => "localhost",
        // ...
    )
);
```

``` php
// index.php
$f3->mset(require('config.php'));
```

Another option would be to use something like this, and still use F3's INI config parser:

``` php
// config.ini.php
;<?php exit(); ?>
[globals]
db.host=localhost
...
```

**Comment by Nokim** [comment]
Adding php code to ini can't help because it is not processed with PHP (at least in nginx config). So, only two choices: hide with web server config (not reliable) or convert to php file.

**Comment by Nokim** [comment]
Third option is move part of project from web-accessible area. Seems not option for some shared hostings...

**Comment by KindlyFire** [comment]
The second option (with a `config.ini.php`) should work fine *everywhere*.

**Comment by Alanaktion** [comment]
This is implemented and will be released in v1.6.
